### PR TITLE
Test case of OrchestrationProperties and NacosProperties

### LINKED ISSUE #4820: Add test cases for TypedProperties implementation
Please add test cases for TypedProperties implementation. The example is `org.apache.shardingsphere.underlying.common.config.properties.ConfigurationPropertiesTest`

The remains test cases need to be finished are: 

- [ ] Test case of OrchestrationProperties
- [x] Test case of ZookeeperProperties
- [x] Test case of EtcdProperties
- [ ] Test case of NacosProperties
- [x] Test case of ApolloProperties

### PR DESCRIPTION
**Pull request by ruanjiehui**
Fixes #4820

Changes proposed in this pull request:
- Test case of OrchestrationProperties 
- Test case of NacosProperties

### CODE REVIEW

File: sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-zookeeper-curator/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ZookeeperPropertiesTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        assertThat(actual.getValue(ZookeeperPropertyKey.TIME_TO_LIVE_SECONDS), is(600));
+        assertThat(actual.getValue(ZookeeperPropertyKey.OPERATION_TIMEOUT_MILLISECONDS), is(6000));
+        assertThat(actual.getValue(ZookeeperPropertyKey.DIGEST), is("any"));
+    }
+
+    @Test
+    public void assertGetDefaultValue() {
+        Properties props = new Properties();
+        ZookeeperProperties actual = new ZookeeperProperties(props);
+        assertThat(actual.getValue(ZookeeperPropertyKey.RETRY_INTERVAL_MILLISECONDS), is(500));
```
Comment (terrymanu) [inline sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-zookeeper-curator/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ZookeeperPropertiesTest.java]: The class ZookeeperPropertiesTest is already existed, why rewrite it again?

Comment (ruanjiehui) [reply sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-zookeeper-curator/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ZookeeperPropertiesTest.java]: @terrymanu  sorry,I submitted pull request based on the last branch, so there is a duplicate file

Comment (ruanjiehui) [reply sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-zookeeper-curator/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ZookeeperPropertiesTest.java]: How should I handle this situation? Is it better to recreate a branch and resubmit the pr after modification or to resolve the conflict?

File: sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-zookeeper-curator/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ZookeeperPropertiesTest.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
 import static org.junit.Assert.assertThat;
 
 public final class ZookeeperPropertiesTest {
-
```
Comment (kimmking) [inline sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-zookeeper-curator/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ZookeeperPropertiesTest.java]: should keep origin blankspaces

File: sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-facade/src/test/java/org/apache/shardingsphere/orchestration/core/facade/properties/OrchestrationPropertiesTest.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    @Test
+    public void assertGetDefaultValue() {
+        Properties props = new Properties();
+        OrchestrationProperties actual = new OrchestrationProperties(props);
+        assertFalse(actual.getValue(OrchestrationPropertyKey.OVERWRITE));
+    }
+
+}
```
Comment (kimmking) [inline sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-facade/src/test/java/org/apache/shardingsphere/orchestration/core/facade/properties/OrchestrationPropertiesTest.java]: let a blank line at tail

File: sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-nacos/src/test/java/org/apache/shardingsphere/orchestration/center/instance/NacosPropertiesTest.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+    @Test
+    public void assertGetDefaultValue() {
+        Properties props = new Properties();
+        NacosProperties actual = new NacosProperties(props);
+        assertThat(actual.getValue(NacosPropertyKey.GROUP), is("SHARDING_SPHERE_DEFAULT_GROUP"));
+        assertThat(actual.getValue(NacosPropertyKey.TIMEOUT), is(3000L));
+    }
+
+}
```
Comment (kimmking) [inline sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-nacos/src/test/java/org/apache/shardingsphere/orchestration/center/instance/NacosPropertiesTest.java]: let a blank line at tail.

**Comment by ruanjiehui** [comment]
@kimmking Thank you for pointing it out,I fix them.

File: sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-facade/src/test/java/org/apache/shardingsphere/orchestration/core/facade/properties/OrchestrationPropertiesTest.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    }
+    
+    @Test
+    public void assertGetDefaultValue() {
+        Properties props = new Properties();
+        OrchestrationProperties actual = new OrchestrationProperties(props);
+        assertFalse(actual.getValue(OrchestrationPropertyKey.OVERWRITE));
+    }
+
+}
```
Comment (kimmking) [inline sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-facade/src/test/java/org/apache/shardingsphere/orchestration/core/facade/properties/OrchestrationPropertiesTest.java]: let an empty line here

**Comment by kimmking** [comment]
@ruanjiehui we should let an empty line at tail.

**Comment by ruanjiehui** [comment]
> @ruanjiehui we should let an empty line at tail.

@kimmking I try to edit file on github website,but found that not to take effect when I add an empty line at tail of file.

**Comment by ruanjiehui** [comment]
When edit on github website,if I need add an empty line at tail of file,I should add two empty lines.

File: sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-nacos/src/test/java/org/apache/shardingsphere/orchestration/center/instance/NacosPropertiesTest.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    }
+    
+    @Test
+    public void assertGetDefaultValue() {
+        Properties props = new Properties();
+        NacosProperties actual = new NacosProperties(props);
+        assertThat(actual.getValue(NacosPropertyKey.GROUP), is("SHARDING_SPHERE_DEFAULT_GROUP"));
+        assertThat(actual.getValue(NacosPropertyKey.TIMEOUT), is(3000L));
+    }
+
```
Comment (terrymanu) [inline sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-nacos/src/test/java/org/apache/shardingsphere/orchestration/center/instance/NacosPropertiesTest.java]: Please remove useless blank line.

File: sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-facade/src/test/java/org/apache/shardingsphere/orchestration/core/facade/properties/OrchestrationPropertiesTest.java [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        assertTrue(actual.getValue(OrchestrationPropertyKey.OVERWRITE));
+    }
+    
+    @Test
+    public void assertGetDefaultValue() {
+        Properties props = new Properties();
+        OrchestrationProperties actual = new OrchestrationProperties(props);
+        assertFalse(actual.getValue(OrchestrationPropertyKey.OVERWRITE));
+    }
+
```
Comment (terrymanu) [inline sharding-orchestration/sharding-orchestration-core/sharding-orchestration-core-facade/src/test/java/org/apache/shardingsphere/orchestration/core/facade/properties/OrchestrationPropertiesTest.java]: Please remove useless blank line.

**Pull request merged**